glTF-Toolkit/inc/GLTFLODUtils.h

```diff
@@ -40,7 +40,7 @@ namespace Microsoft::glTF::Toolkit
         /// <param name="docs">A vector of glTF documents to merge as LODs. The first element of the vector is assumed to be the primary LOD.</param>
         /// <param name="screenCoveragePercentages">A vector with the screen coverage percentages corresponding to each LOD. If the size of this 
         /// vector is larger than the size of <see name="docs" />, lower coverage values will cause the asset to be invisible.</param>
-        static GLTFDocument MergeDocumentsAsLODs(const std::vector<GLTFDocument>& docs, const std::vector<double>& screenCoveragePercentages);
+        static GLTFDocument MergeDocumentsAsLODs(const std::vector<GLTFDocument>& docs, const std::vector<std::wstring>& relativePaths, const std::vector<double>& screenCoveragePercentages);

```

Comment:
This should have an option without this parameter that assumes everything is in the same directory